### DISCUSSION TITLE
ΠΑΡΑΤΑΣΗ ΠΑΡΑΔΟΣΗΣ ΓΕ2

### DISCUSSION THREAD

**Discussion by gregtsag** [sdy60/syllabus#52] [Announcements]
Καλησπέρα,

Σχετικά με την παράταση 7 ημερών που είχαμε αναφέρει στην τελευταία ΟΣΣ θα δωθεί οριζόντια ή να ζητήσουμε επι μέρους;

**Comment by epidrome** [comment]
αν συμφωνούν όλοι τότε οριζόντια παράταση μιας εβδομάδας σε όλους μέχρι Τετ. 21 Δεκ.

**Reply by Katethoma** [reply]
Εγώ συμφωνώ με την οριζόντια παράταση

**Comment by vkaragia** [comment]
οκ από εμένα
